src/api.ts

```diff
@@ -17,7 +17,7 @@ function get<T>(url: string): Promise<T> {
   const fullURL = BASE_URL + url + '.json';
 
   // we add some delay to see how the loader works
-  return wait(300)
+  return wait(1)
```

Comment:
i don't understand why...

Reply:
@MarcinLigmann to make wait time shorter, basically `wait` is to simulate a network request